components/Player/NotesTable.tsx

```diff
@@ -1,17 +1,39 @@
 import { Notes } from "@prisma/client";
 import Button from "components/Button";
-import React from "react";
+import { ReadManyNotesDTO } from "pages/api/notes/readMany";
+import React, { Fragment, useEffect, useState } from "react";
+import { Menu, Transition } from "@headlessui/react";
+import Icon from "components/Icon";
+
+interface Category {
+  color: string;
+  enabled: boolean;
+}
+
+const CATEGORIES = {
+  General: "blue-muted",
+  Soccer: "purple-muted",
+  Academics: "pink-muted",
+  Mentorship: "gold-muted",
+};
 
 const Note: React.FunctionComponent<{ note: Notes }> = ({
   // eslint-disable-next-line camelcase
   note: { created_at, content, authorId, type },
 }) => {
   return (
-    <div className=" grid grid-rows-3 text-sm max-h-56 border-opacity-50 border-b">
-      <text className="row-span-3 w-16 h-4 text-xs text-center rounded-full font-semibold text-unselected bg-button mt-10">
-        {type}
+    <div className=" grid grid-rows-3 grid-cols-3 text-sm max-h-56 border-opacity-50 border-b">
+      <text
```

Comment:
It looks like for Mentorship and Academics the tag background it too short do you think we can fix this?